<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>654</width>
    <height>377</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ScrCap - screen capture</string>
  </property>
  <property name="windowIcon">
   <iconset theme="video-display">
    <normaloff>.</normaloff>.</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Capture</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QFrame" name="frame">
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout">
             <item>
              <widget class="QLabel" name="imageDisplay">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <widget class="QPushButton" name="btnCapture">
              <property name="text">
               <string>Capture</string>
              </property>
              <property name="icon">
               <iconset theme="view-preview">
                <normaloff>.</normaloff>.</iconset>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btnAutocapture">
              <property name="text">
               <string>Autocapture</string>
              </property>
              <property name="icon">
               <iconset theme="tools-wizard">
                <normaloff>.</normaloff>.</iconset>
              </property>
              <property name="checkable">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="verticalSpacer_4">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QComboBox" name="listMode"/>
            </item>
            <item>
             <spacer name="verticalSpacer">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="btnSave">
              <property name="text">
               <string>Save as...</string>
              </property>
              <property name="icon">
               <iconset theme="document-save">
                <normaloff>.</normaloff>.</iconset>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btnCopy">
              <property name="text">
               <string>Copy to clipboard</string>
              </property>
              <property name="icon">
               <iconset theme="edit-copy">
                <normaloff>.</normaloff>.</iconset>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btnClose">
              <property name="text">
               <string>Close</string>
              </property>
              <property name="icon">
               <iconset theme="dialog-close">
                <normaloff>.</normaloff>.</iconset>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Capture &amp;delay</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
            <property name="buddy">
             <cstring>spinDelay</cstring>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="spinDelay">
            <property name="specialValueText">
             <string>without delay</string>
            </property>
            <property name="suffix">
             <string> sec</string>
            </property>
            <property name="maximum">
             <number>9999</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Parameters</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_7">
        <item>
         <widget class="QGroupBox" name="groupBox_2">
          <property name="title">
           <string>Capture options</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_6">
             <item>
              <widget class="QCheckBox" name="checkIncludeDeco">
               <property name="text">
                <string>Include window decorations</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="checkIncludePointer">
               <property name="text">
                <string>Include mouse pointer</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_5">
             <item>
              <layout class="QFormLayout" name="formLayout">
               <item row="0" column="0">
                <widget class="QLabel" name="label_4">
                 <property name="text">
                  <string>&amp;Interactive capture hotkey</string>
                 </property>
                 <property name="buddy">
                  <cstring>keyInteractive</cstring>
                 </property>
                </widget>
               </item>
               <item row="0" column="1">
                <widget class="QKeySequenceEdit" name="keyInteractive">
                 <property name="toolTip">
                  <string>You can also bring ScrCap window back from autocapture/silent session by pressing this hotkey.</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="0">
                <widget class="QLabel" name="label_6">
                 <property name="text">
                  <string>Silent capture &amp;&amp; save &amp;hotkey</string>
                 </property>
                 <property name="buddy">
                  <cstring>keySilent</cstring>
                 </property>
                </widget>
               </item>
               <item row="1" column="1">
                <widget class="QKeySequenceEdit" name="keySilent"/>
               </item>
               <item row="2" column="0">
                <widget class="QLabel" name="label_5">
                 <property name="text">
                  <string>&amp;Autocapture scan interval</string>
                 </property>
                 <property name="buddy">
                  <cstring>spinAutocapInterval</cstring>
                 </property>
                </widget>
               </item>
               <item row="2" column="1">
                <widget class="QSpinBox" name="spinAutocapInterval">
                 <property name="toolTip">
                  <string>Also DE notification timeout</string>
                 </property>
                 <property name="suffix">
                  <string> msec</string>
                 </property>
                 <property name="minimum">
                  <number>50</number>
                 </property>
                 <property name="maximum">
                  <number>3600000</number>
                 </property>
                 <property name="singleStep">
                  <number>50</number>
                 </property>
                 <property name="value">
                  <number>1000</number>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QFormLayout" name="formLayout_2">
               <item row="0" column="0">
                <widget class="QLabel" name="label_7">
                 <property name="text">
                  <string>I&amp;mage format</string>
                 </property>
                 <property name="buddy">
                  <cstring>listImgFormat</cstring>
                 </property>
                </widget>
               </item>
               <item row="0" column="1">
                <widget class="QComboBox" name="listImgFormat"/>
               </item>
               <item row="1" column="0">
                <widget class="QLabel" name="label_8">
                 <property name="text">
                  <string>Image &amp;quality</string>
                 </property>
                 <property name="buddy">
                  <cstring>spinImgQuality</cstring>
                 </property>
                </widget>
               </item>
               <item row="1" column="1">
                <widget class="QSpinBox" name="spinImgQuality">
                 <property name="maximum">
                  <number>100</number>
                 </property>
                 <property name="value">
                  <number>90</number>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox">
          <property name="title">
           <string>Files</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_6">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item>
              <widget class="QLabel" name="label_2">
               <property name="text">
                <string>&amp;Save directory</string>
               </property>
               <property name="buddy">
                <cstring>editDir</cstring>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="editDir"/>
             </item>
             <item>
              <widget class="QPushButton" name="btnDir">
               <property name="text">
                <string>...</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_9">
             <item>
              <widget class="QLabel" name="label_10">
               <property name="text">
                <string>A&amp;utocapture sound</string>
               </property>
               <property name="buddy">
                <cstring>editAutoSnd</cstring>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="btnSndPlay">
               <property name="text">
                <string/>
               </property>
               <property name="icon">
                <iconset theme="media-playback-start"/>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="editAutoSnd"/>
             </item>
             <item>
              <widget class="QPushButton" name="btnAutoSnd">
               <property name="text">
                <string>...</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <item>
              <widget class="QLabel" name="label_3">
               <property name="text">
                <string>&amp;Filename template
(without extension)</string>
               </property>
               <property name="buddy">
                <cstring>editTemplate</cstring>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="editTemplate">
               <property name="toolTip">
                <string>%NN - counter
%w - width
%h - height
%y - year
%m - month
%d - day
%t - time</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>tabWidget</tabstop>
  <tabstop>btnCapture</tabstop>
  <tabstop>btnAutocapture</tabstop>
  <tabstop>listMode</tabstop>
  <tabstop>btnSave</tabstop>
  <tabstop>btnCopy</tabstop>
  <tabstop>btnClose</tabstop>
  <tabstop>spinDelay</tabstop>
  <tabstop>checkIncludeDeco</tabstop>
  <tabstop>checkIncludePointer</tabstop>
  <tabstop>keyInteractive</tabstop>
  <tabstop>keySilent</tabstop>
  <tabstop>spinAutocapInterval</tabstop>
  <tabstop>listImgFormat</tabstop>
  <tabstop>spinImgQuality</tabstop>
  <tabstop>editDir</tabstop>
  <tabstop>btnDir</tabstop>
  <tabstop>btnSndPlay</tabstop>
  <tabstop>editAutoSnd</tabstop>
  <tabstop>btnAutoSnd</tabstop>
  <tabstop>editTemplate</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>btnClose</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>631</x>
     <y>322</y>
    </hint>
    <hint type="destinationlabel">
     <x>610</x>
     <y>305</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
